### DISCUSSION TITLE
modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/LiquidRegistry

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#104] [Announcements]
# modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/LiquidRegistry

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/LiquidRegistry

**Comment by JasonQ1123** [comment]
流体流动速度可以这么写，我提供一下代码
    e.create("createdelight:whipped_cream")
        .stillTexture("createdelight:fluid/whipped_cream/still")
        .flowingTexture("createdelight:fluid/whipped_cream/flowing")
        .translationKey("fluid.createdelight.whipped_cream")
        .translationKey("block.createdelight.whipped_cream")
        .translationKey("item.createdelight.whipped_cream_bucket")
        .createAttributes()
        .tickDelay(10)
tickDelay里面的数值代表延缓速度（流动的时候的延缓速度）
